lib/components/Field/FormError.tsx

```diff
@@ -30,12 +33,18 @@ export const FormError: React.StatelessComponent<FormErrorProps> = (props: FormE
     return (
         <Attr.div
             className={css('field-error', {
-                'hidden': props.hidden
+                hidden: props.hidden
             }, props.className)}
             title={props.title}
             attr={props.attr.container}
         >
-            {props.children}
+            {props.children 
+                ? <>
+                    {!props.hideIcon && <Icon icon='errorBadge' size={IconSize.small} className={css('error-badge')} />}
+                    <span className={css('inline-text-overflow')}>{props.children}</span>
```

Comment:
why add `inline-text-overflow` was this behavior already there? Is anyone else setting this class to the error?

Reply:
the behavior was not there before but it should have been. without this addition, the text is not properly clipped if it overflows